compiler/compile/OMRCompilation.cpp

```diff
@@ -935,19 +935,20 @@ int32_t OMR::Compilation::compile()
       self()->setOption(TR_DisablePartialInlining);
 
 #ifdef J9_PROJECT_SPECIFIC
-   if (self()->getOptions()->getDelayCompile())
+   if (self()->getOptions()->getDelayCompileWithCPUBurn())
       {
-      uint64_t limit = (uint64_t)self()->getOptions()->getDelayCompile();
+      uint64_t limit = (uint64_t)self()->getOptions()->getDelayCompileWithCPUBurn();
       uint64_t starttime = self()->getPersistentInfo()->getElapsedTime();
-      fprintf(stderr,"\nDelayCompile: Starting a delay of length %" OMR_PRIu64 " for method %s at time %" OMR_PRIu64 ".", limit, self()->signature(), starttime);
-      fflush(stderr);
+      if (self()->getOptions()->getVerboseOption(TR_VerbosePerformance))
+         TR_VerboseLog::writeLineLocked(TR_Vlog_PERF, "DelayCompile: Starting a delay of length %" OMR_PRIu64 " for method %s at time %" OMR_PRIu64 ".", limit, self()->signature(), starttime);
```

Comment:
How about changing the message prefix from `DelayCompile:` to `DelayCompileWithCPUBurn:` ?

Reply:
Done